index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Facebook</title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <script src="https://ajax.aspnetcdn.com/ajax/jQuery/jquery-3.3.1.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.11/handlebars.js"></script>
+    </head>
+    <body>
+        <h1 id="faceheading">Facebook</h1>
+        <div class="post">
+            <ul id="postList"></ul>
+        </div>
+        <script type="text/javascript" src="script.js"></script>
```

Comment:
why script is inside body ?

Reply:
It's at the bottom of body because I don't want it to load before other html elements.

Reply:
Is the body tag (structure) good for scripts ? maybe put it outside the body - at the end of html document ?